lib/bitmap.c

```diff
@@ -109,16 +109,30 @@ M0_INTERNAL bool m0_bitmap_get(const struct m0_bitmap *map, size_t idx)
 }
 M0_EXPORTED(m0_bitmap_get);
 
-M0_INTERNAL size_t m0_bitmap_ffz(const struct m0_bitmap *map)
+M0_INTERNAL int m0_bitmap_ffs(const struct m0_bitmap *map)
 {
-	size_t idx;
+	int i;
+	int idx;
+
+	for (i = 0; i < M0_BITMAP_WORDS(map->b_nr); i++) {
+		idx = __builtin_ffsll(map->b_words[i]);
```

Comment:
Is this GCC built-in function?
If we want to port to other platform, e.g. ARM, is it still available?